pkg/provider/resource_region.go

```diff
@@ -4,185 +4,243 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-framework/path"
```

Comment:
If I'm not mistaken, this should be added to the down (in the following lines of the import statement)

Reply:
right, rerun `goimports`